.dockerignore

```diff
@@ -1,4 +1,7 @@
 node_modules
 npm-debug.log
 .idea
-.elasticbeanstalk
\ No newline at end of file
+.elasticbeanstalk
```

Comment:
If you see the `Dockerfile` the whole build context is added to the image using `COPY . /usr/src/app`. But the listed directories are not part of the NodeJS application, and some of them are build contexts to some other images as well. This would result in unwanted files getting stored in the final image. Adding these folders to the `.dockerignore` will make sure that only the NodeJS code is added to the final image, making it thinner as well as secure.